### PR TITLE
docs: pwa-ify docs site

### PR DESCRIPTION
**Pull request by daKmoR**

### CODE REVIEW

**Comment by daKmoR** [comment]
@thepassle couldn't easily fix the history of your branch so I cherry picked it... does this look still good?

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -155,9 +155,26 @@ module.exports = {
...
   },
   dest: '_site',
-  plugins: ['@vuepress/google-analytics'],
+  plugins: [
+    '@vuepress/google-analytics',
+    [
+      '@vuepress/pwa',
+      {
+        serviceWorker: true,
+        updatePopup: false,
```
Comment (daKmoR) [inline docs/.vuepress/config.js]: so how will new content happen then?

Comment (thepassle) [reply docs/.vuepress/config.js]: It uses skipwaiting, see: https://github.com/open-wc/open-wc/pull/1053#issuecomment-559741070

**Comment by thepassle** [comment]
Yep, LGTM :)

**Comment by daKmoR** [comment]
awesome - so we have full offline support now 🎉 🎉

**Pull request merged**